src/ApiTestCase.php

```diff
@@ -51,10 +51,11 @@ abstract class ApiTestCase extends WebTestCase
     /** @var EntityManager|null */
     private $entityManager;
 
-    public function __construct(?string $name = null, array $data = [], $dataName = '')
+    /**
+     * @before
```

Comment:
```suggestion
     * @beforeClass
```

As matcher should probably be initiated only once